thermal-recorder-TEMPLATE.yaml

```diff
@@ -27,25 +30,41 @@ min-disk-space: 200
 motion:
     # Movement below raw temperatures of this value will not activate
     # motion detection.
-    temp-thresh: 3000
+    temp-thresh: 2900
 
     # Minimum raw temperature difference between recent frames to
     # trigger motion detection.
-    delta-thresh: 30
+    delta-thresh: 50
 
     # Number of pixels which must show delta-thresh change before
     # motion detection event will be triggered.
-    count-thresh: 5
+    count-thresh: 3
 
     # Suppress motion detection if more than this percentage of the
     # frame appears to change. This helps avoid false positives when
     # the camera is recalibrating.
     nonzero-max-percent: 50
 
-# GPIO pins for the LEDs
-leds:
-    # Pin of the blue LED, indicating that the device is recording.
-    recording: "GPIO20"
+    # When working out which pixels have changed it tries to compare
+    # the current frame with one recorded this many frames ago. If there
+    # aren't this many frames (eg since a recalibration) then it will use
+    # the oldest frame it has after the last 'bad' frame.
+    FrameCompareGap: 45
+
+    # When working out if motion has occured only look at a single frame diff.
+    # If set to false then it looks as two sequential frame diffs and only consideres
+    # pixels that have changed in both diffs
+    # Note: set to false because rats move too fast!
```

Comment:
I don't understand this note. It seems to be saying that it is preferable to have this setting set to false, but the default is true.